core/src/test/java/org/hibernate/ogm/backendtck/queries/parameters/QueryWithParametersTest.java

```diff
@@ -35,7 +35,7 @@
 	public PackagingRule packaging = new PackagingRule( "persistencexml/ogm.xml", Movie.class );
 
 	@Test
-	@SkipByGridDialect(value = { INFINISPAN_REMOTE }, comment = "Query on Byte field are currently not supported")
+	@SkipByGridDialect(value = { INFINISPAN_REMOTE }, comment = "Query on --starting with 'v'-- field are currently not supported")
```

Comment:
This looks weird. What does it mean?

Reply:
A strange reference to ISPN-7863 , maybe :))))

Reply:
Hum, OK. AFAICS, it's fixed in 9.3.4.Final so I don't think it's worth it to reference this ISPN bug in our code.

Reply:
Thank you @gsmet and @anistor. I will change the comment here.